
---
string[] entitiesNames
uint64 length
